modules/apps/transfer/keeper/relay_test.go

```diff
@@ -394,10 +374,7 @@ func (suite *KeeperTestSuite) TestOnRecvPacket_ReceiverIsNotSource() {
 			data := types.NewFungibleTokenPacketDataV2(
 				[]types.Token{
 					{
-						Denom: types.Denom{
-							Base:  sdk.DefaultBondDenom,
-							Trace: []types.Trace{},
-						},
+						Denom:  types.NewDenom(sdk.DefaultBondDenom),
```

Comment:
I don't know if this requires it or not, but this now changed from empty slice

```suggestion
						Denom:  types.NewDenom(sdk.DefaultBondDenom, []types.Trace{}),
```